tree/ntuple/v7/test/ntuple_join_table.cxx

```diff
@@ -136,12 +136,13 @@ TEST(RNTupleJoinTable, SparseSecondary)
       auto event = fldEvent(i);
 
       if (i % 2 == 1) {
-         EXPECT_EQ(joinTable->GetFirstEntryNumber<std::uint64_t>(event), ROOT::kInvalidNTupleIndex)
+         EXPECT_EQ(joinTable->GetEntryIndexes({&event}), std::vector<ROOT::NTupleSize_t>{})
             << "entry should not be present in the join table";
       } else {
-         auto idx = joinTable->GetFirstEntryNumber<std::uint64_t>(event);
-         EXPECT_EQ(idx, i / 2);
-         EXPECT_FLOAT_EQ(fldX(idx), static_cast<float>(idx) / 3.14);
+         auto entryIdxs = joinTable->GetEntryIndexes({&event});
+         EXPECT_EQ(1ul, entryIdxs.size());
```

Comment:
You may want to use `ASSERT_EQ` here (and elsewhere) to immediately fail the test instead of accessing out-of-bounds below